nikola/__main__.py

```diff
@@ -64,6 +64,25 @@
 # DO NOT USE unless you know what you are doing!
 _RETURN_DOITNIKOLA = False
 
+PY2_WARNING = """Nikola is going to deprecate Python 2 support in 2016. You already have Python 3
```

Comment:
I actually intended to check if it's available, and use a different message if it's  not.